US County Data - pandas.ipynb

```diff
@@ -7,7 +7,10 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    question1 = us_counties_df.reset_index()
</pre><p>Don't need the reset_index</p><p> </p><p>New York City is not a county, hence the FIPS code is NaN</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CMU-MS-DAS-Modern-Programming-Mini/ClassAssignmentNo4/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>